src/pydata_sphinx_theme/__init__.py

```diff
@@ -511,9 +511,9 @@ def navbar_align_class():
         """Return the class that aligns the navbar based on config."""
         align = context.get("theme_navbar_align", "content")
         align_options = {
-            "content": ("col-lg-9", "mr-auto"),
-            "left": ("", "mr-auto"),
-            "right": ("", "ml-auto"),
+            "content": ("col-lg-9", "me-auto"),
+            "left": ("", "me-auto"),
+            "right": ("", "ms-auto"),
```

Comment:
not thoroughly checked, but PR looks the same as `main`

Reply:
on main we use `ml` and `mr` but here I use `ms` and `me

> Horizontal direction specific variables, utilities, and mixins have all been renamed to use logical properties like those found in flexbox layouts—e.g., start and end in lieu of left and right.